index.html

```diff
@@ -7,177 +7,352 @@
     <link rel="stylesheet" href="styles.css">
   </head>
 
-  <body>
-    <header>
-      <a id="logo">Mark.O</a>
-      <nav>
-        <img src="./logos/Union.jpeg" alt="menue bar" id="menue-button">
+  
+<body>
+  <div class="hero">
```

Comment:
- [ ] [REQUESTED] kindly as requested by the previous reviewer, fix this issue by removing the `<div class="hero">` because we're following the HTML semantics it means the browser will know when to layout the header. please do that and let see you project get approved :handshake:

Reply:
just the hero class yh?